website/modules/seo.ts

```diff
@@ -0,0 +1,29 @@
+export default function seoModule () {
```

Comment:
It is best if you keep this in the main `nuxt.config`. So we can migrate to [this RFC](https://github.com/nuxt/framework/discussions/1823) when implemented.

Reply:
@pi0 done!